libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/OfflineTileVersions.kt

```diff
@@ -0,0 +1,47 @@
+package com.mapbox.navigation.route.onboard
+
+import com.mapbox.api.routetiles.v1.versions.MapboxRouteTileVersions
+import com.mapbox.api.routetiles.v1.versions.models.RouteTileVersionsResponse
+import com.mapbox.navigation.route.onboard.model.OfflineError
+import retrofit2.Call
+import retrofit2.Callback
+import retrofit2.Response
+
+/**
+ * This is a wrapper class for the [MapboxRouteTileVersions] class. It returns a list of
+ * all available versions of Routing Tiles available via [OfflineTiles]. This class
+ * encapsulates the unwrapping of the list from the response.
+ */
+internal class OfflineTileVersions {
+
+    /**
+     * Call to receive all the available versions of Offline Tiles available.
+     *
+     * @param accessToken for the API call
+     * @param callback to be updated with the versions
+     */
+    fun fetchRouteTileVersions(accessToken: String, callback: OnTileVersionsFoundCallback) {
```

Comment:
This is currently implemented but never used. As mentioned above, `MapboxOnboardRouter` is missing some APIs.